config-linux.md

```diff
@@ -494,6 +494,31 @@ You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given ent
 }
 ```
 
+## <a name="configLinuxUnified" />Unified
+
+**`unified`** (object, OPTIONAL) allows cgroup v2 parameters to be to be set and modified for the container.
+
+Each key in the map refers to a file in the cgroup unified hierarchy.
```

Comment:
I understand not binding the to kernel API,  but should we link to docs and note that these names will change/add per the kernel versions. Also denote any error handling for unknown seettings?

Reply:
sure I think we can add a note saying these files can change per kernel version.

The rationale for having a map here is that the OCI runtime doesn't have to know about what the settings mean as long as it writes the value to the specified file.  Should the error handling be anything more than propagating the errno from `open` or `write`?

Reply:
> sure I think we can add a note saying these files can change per kernel version.
> 
> The rationale for having a map here is that the OCI runtime doesn't have to know about what the settings mean as long as it writes the value to the specified file. Should the error handling be anything more than propagating the errno from `open` or `write`?

The implementation also needs to make sure the corresponding controller is available and enabled (ie. recursively write to `cgroup.subtree_control`). The controller can be unknown to the implementation, not be available (due to being used in cgroup1 or not being complied into the kernel) or lack features (eg. `io.bfq.*`).

Reply:
isn't that implicit with `The runtime MUST generate an error when the configuration refers to a cgroup controller that is not present or that cannot be enabled.`.

Even if the controller is not present because not compiled in the kernel, the runtime MUST fail and return an error if the configuration specified a setting that cannot be honored.